tests/src/test/scala/scalaz/MapTest.scala

```diff
@@ -640,6 +794,120 @@ object MapTest extends SpecLite {
       val f = (k: Int, a: String, result: String) => result + "(" + k.toString + ":" + a + ")"
       fromList(List(5 -> "a", 3 -> "b")).foldrWithKey("Map: ")(f) must_== "Map: (5:a)(3:b)"
     }
+
+    "foldMapWithKey" ! forAll { a: Byte ==>> Byte =>
+      val intAddition = new Monoid[Int] {
```

Comment:
We can use default `intInstance: Monoid[Int]` in `AnyVal.scala`.
https://github.com/scalaz/scalaz/blob/v7.3.0-M2/core/src/main/scala/scalaz/std/AnyVal.scala#L234-L237

I think we should just remove `intAddition` in this file.